uint32 gACT = 1
uint32 gGTO = 2
uint32 gSTA = 3
uint32 gOBJ = 4
uint32 gFLT = 5
uint32 gPR =6
uint32 gPO =7
uint32 gCU =8
